src/MensaWidget/MWFrameTest.class.st

```diff
@@ -0,0 +1,83 @@
+"
+A MWFrameTest is Test Case for the MWFrame.
+
+Instance Variables
+	widgetFrame:		<MWFrame>
+
+widgetFrame
+	- a instance of the MWFrame on which the tests will be done
+
+"
+Class {
+	#name : #MWFrameTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'widgetFrame'
+	],
+	#category : #'MensaWidget-Tests',
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:44'
+}
+
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 17:12'
+}
+MWFrameTest >> setUp [
+
+	widgetFrame := MWFrame new
+]
+
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 17:10'
+}
+MWFrameTest >> tearDown [
+
+	widgetFrame abandon
```

Comment:
if we agree on making accessors, we should use them here

Reply:
done